app/services/mentions_create_all_service.rb

```diff
@@ -0,0 +1,43 @@
+class MentionsCreateAllService
```

Comment:
I suggest putting a service inside a namespace as it already was created for the job. the `Service` seems a bit redundant to me.